### ISSUE TITLE
Error on zsh shell launch

### ISSUE THREAD

**Issue by noahgaertner** [zdharma/zinit#535]
**Describe the bug**
On launch, 
```zsh
~                                                          at 06:16:25 PM
❯
/Users/noahgaertner/.zinit/plugins/_local---lazyenv/evalcache.zsh:source:14: no such file or directory: /opt/homebrew/Cellar/pyenv/2.0.3/libexec/../completions/pyenv.zsh
~                                                          at 06:16:25 PM
❯
```
**To Reproduce**
honestly no idea - I don't have pyenv installed (or even listed anywhere in my zshrc, I checked) - and the mentioned file is completely blank (as in there is nothing in it at all).
**Expected behavior**
not have that error show up
**Versions:**
 - $ZSH_VERSION == zsh 5.8 (arm-apple-darwin20.2.0)
 - Operating system name and version (uname -a): MacOS 12.0 Monterey Public Beta 4

**Comment by fugkco** [comment]
It looks like you've got something called maybe lazyenv? Do an `rm -rf /Users/noahgaertner/.zinit/plugins/_local---lazyenv` (or mv it instead to be safe) and restart a new zsh instance, and see what the output is.

**Comment by fugkco** [comment]
Yeah, check if you have [this](https://github.com/black7375/zsh-lazyenv) installed. It seems to be trying to load some pyenv related stuff, which it is unable to find.
